### DISCUSSION TITLE
Is there a way to hide BPM marker, tuning and instrument name?

### DISCUSSION THREAD

**Discussion by bali182** [CoderLine/alphaTab#1594] [Q&A] [answered]
I'd like to hide the BPM marker, the tuning and the small instrument name. Is it possible somehow using the API?
![Screenshot 2024-07-23 184023](https://github.com/user-attachments/assets/494a0223-c422-4cab-8232-9ff3dda31c95)

**Comment by Danielku15** [comment, marked as answer]
Sorry for the delayed answer. Element visibility can be controlled through https://www.alphatab.net/docs/reference/settings/notation/elements/

**Reply by bali182** [reply]
Fantastic, thank you very much!
